content/docs/start/experiments-trail/experiments.md

```diff
@@ -0,0 +1,319 @@
+```
+
+The command tells DVC to create an experiment named `train`, and that for any
+change in `data/images/`, `model.conv_units` or `train.epochs`, we run an
+experiment using `src/train.py` that produces a new `metrics.json` file.
+
+Note that the parameters (added with `-p`) are in the default parameters file
+`params.yaml` and used in the code as normal, by reading the file. DVC only
+tracks the changes and updates them with `--set-param`. For other files that
+contain the parameters, DVC expects the filename to be provided similar to
+`-p myparams.yaml:epochs`.
+
+DVC is ready to run the experiments now!
+
+</details>
+
+## 👟 Running the experiment with default parameters
+
+The purpose of `dvc exp` subcommands is to run the pipeline for ephemeral
+experiments. By _ephemeral_ we mean the experiments can be run without
+committing parameter and dependency changes to Git. Instead the artifacts
+produced for each experiment are tracked by DVC and persisted on demand.
+
+Running the experiment with default project settings requires only the command:
+
+```dvc
+$ dvc exp run
+...
+Reproduced experiment(s): exp-7683f
+Experiment results have been applied to your workspace.
+...
+```
+
+It runs the specified command that writes the metrics values to `metrics.json`.
+
+<details>
+
+### 📜 If you used `dvc repro` before
+
+Earlier versions of DVC uses `dvc repro` to run the pipeline. If you already
+have a DVC project, you may already be using `dvc repro`.
+
+In DVC 2.0 `dvc exp run` supersedes `dvc repro`. Both of these commands run the
+pipeline.
+
+We use `dvc repro` to run the pipeline as found in the <abbr>workspace</abbr>.
+All the parameters and dependencies are retrieved from the current workspace. It
+doesn't use any special objects to track the experiments. When you have large
+number of experiments that you don't want to commit all to Git, it's better to
+use `dvc exp run`. It allows to change the parameters quickly, can track the
+history of artifacts and has facilities to compare these experiments easily.
+
+`dvc repro` is still available to run the pipeline when these extra features are
+not needed.
+
+</details>
+
+## 🧥 Running the experiment by setting parameters
+
+Now let's do some more experimentation.
+
+DVC allows to update the parameters defined in the pipeline without modifying
+the files manually. We use this feature to set the convolutional units in
+`train.py`.
+
+```dvc
+$ dvc exp run --set-param model.conv_units=24
+...
+Reproduced experiment(s): exp-6c06d
+Experiment results have been applied to your workspace.
+
+To promote an experiment to a Git branch run:
+
+        dvc exp branch <exp>
+```
+
+When you run `dvc exp run` with `--set-param`, it updates the parameter file. We
+can see the effect of it by looking at the diff.
+
+```dvc
+$ git diff params.yaml
+```
+
+```git
+-model:
+-  conv_units: 16
++model:
++  conv_units: 24
+```
+
+## 🏃‍♂️🏃🏾‍♂️🏃🏻‍♂️ Run multiple experiments in parallel
```

Comment:
Can we merge three different "running experiments sections" into one or at least to? Otherwise it becomes more of a user guide I think. No intention here to be very comprehensive, we need to make it clear what is possible, ideally with one single command, one single table

Reply:
I rather would like to keep the "sales pitch" in these docs, and IMO that's "running experiments in parallel" for `dvc exp run`. But in order to tell "you can run experiments in parallel", we first need to tell "you can run experiments".

I think keeping these in separate sections emphasize _multiple experiments_. If I merge these, `--queue` option will become a minor thing in a deeper paragraph.

Reply:
I think keeping the "sales pitch" could mean having a compelling header of a hidden section or otherwise calling out the idea, but keeping how to do it in the hidden section. You can still have all these experiments in the `dvc exp show` table even if they aren't in the main text. Having that table higher up seems important since it's unclear how `dvc exp run` is helpful until that point.

Reply:
I put the sections in `details`. Let's see how it looks.